QUEST_LV_0200.tsv

```diff
@@ -1026,40 +1026,40 @@ QUEST_LV_0200_20150317_001025	{memo X}If it weren't you, we would have never got
 QUEST_LV_0200_20150317_001026	{memo X}Um? Isn't this royal jellly?{nl}Mikolas.. It is so fortunate. I was able to get rid of the misunderstanding because of you.{nl}May the Austeja bless you.
 QUEST_LV_0200_20150317_001027	Maras
 QUEST_LV_0200_20150317_001028	{memo X}Was it you who Jozef was looking for? {nl}I don't care if you're not. {nl}If you don't help me, you will be the same as those Revelators who ran away.{nl}
-QUEST_LV_0200_20150317_001029	Anyway, I trusted the priests, but I can't stand anymore.{nl}That's why I want to hire someone who is skillful like you. {nl}I will compensate you enough.
-QUEST_LV_0200_20150317_001030	Anyone would know that it's because of the honey, but I don't know know what the priests are doing.
-QUEST_LV_0200_20150317_001031	I made a request at Klaipeda, but they are not responding since they have a lack of soldiers.{nl}And it is not easy to hire merceneries in this era.{nl}
-QUEST_LV_0200_20150317_001032	Then the ones that are left are the Revelators.{nl}You looking for the Goddess? we have the Goddess in our village too.
-QUEST_LV_0200_20150317_001033	Look carefully on the unbroken bee boxes.{nl}You could maybe obtain some bee boxes that are in good status.
-QUEST_LV_0200_20150317_001034	Oh, this is quite big.{nl}Well done.
-QUEST_LV_0200_20150317_001035	You go back to Bulbini Farm.{nl}If we could provide a proof that it was due to the honey, the priests would not only think about the altar.{nl}
-QUEST_LV_0200_20150317_001036	There are many altars around here.{nl}You know it already right? They are all created to praise Goddess Austeja.
+QUEST_LV_0200_20150317_001029	$Anyways, I've placed all my faith on the priest, but she's quickly losing that faith.{nl}That's why I want to hire someone who is skillful like you. {nl}I will compensate you well.
```

Comment:
Anyways, I've placed all of my faith in the priest, but she's quickly making me lose that faith.